changelog/unreleased/pr-13970.toml

```diff
@@ -0,0 +1,4 @@
+type = "add"
+message = "Dynamic starting page FE"
```

Comment:
Let's add a more meaningful change log, like "Replacing getting started guide with start page which lists recently opened and favorite saved searches and dashboards and recent activity."